backend-models/service-tests/showDuggaservice-test.txt

```diff
@@ -0,0 +1,393 @@
+showDuggaservice.php
+
+—------------------------------------------------------------------------------------------------------------------------
+TEST #1 - 100-105
+Get active users
+
+***
+Pre-req: -
+***
+Send parameters inputs to showDuggaservice.php
+    send
+    {
+        $opt = “UPDATEAU”
+        $hash = hash
+
+    }
+
+    SQL-query: SELECT active_users FROM groupdugga WHERE hash=hash
+    This should show all active users
+
+***
+Gather service output
+***
+    Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+    This is the expected output for the micro service.
+
+—------------------------------------------------------------------------------------------------------------------
+TEST #2 - 106-110
+Create active users
+
+***
+Pre-req: $active == NULL
+***
+
```

Comment:
i dont know how the variables $hash or $AUtoken get there values so this might be right.
but you have to make sure that this value isn't already in the database.

Reply:
We wrote the hash and AUtoken as an example and they work in the database.

Reply:
yes, once again. it works now but can you guarantee that this always will be the case?
you base this on the data currently registered in the database. Presume now that we create a specific test database, only used for test cases. Can we at this moment guarantee that no such value will be added? the chance is pretty, low but there is a chance..

Reply:
solved